extensions/wikia/AssetsManager/config.php

```diff
@@ -776,7 +782,10 @@
 		// Advertisement libs
 		'//extensions/wikia/AdEngine/js/provider/factoryWikiaGpt.js',
 		'//extensions/wikia/AdEngine/js/provider/directGptMaps.js',
-		'//extensions/wikia/AdEngine/js/GptHelper.js',
+		'//extensions/wikia/AdEngine/js/provider/gpt/adElement.js',
+		'//extensions/wikia/AdEngine/js/provider/gpt/adSizeConverter.js',
+		'//extensions/wikia/AdEngine/js/provider/gpt/googleTag.js',
```

Comment:
You changed the file name here but you [didn't rename it](https://github.com/Wikia/app/pull/7565/files#diff-d054b39e41980d65768a4aea15c18288R1). Can you do it? I think the same you can do with the unit tests' file: rename it from `GoogleTag.spec.js` -> `googleTag.spec.js` in sake of consistency :)